docs/_includes/tk-alias

```diff
@@ -1 +1 @@
-2019 — 2022
\ No newline at end of file
+2019 — 2023
```

Comment:
Ok for the new version but should we not increment our "oldest supported" version? (according to our "current + 2 previous ones" rule)